### PR TITLE
Migrate fully to new test naming scheme

### PR DESCRIPTION
**Pull request by ryanwilson**
## Description of the change
Move away from old test input files.

## Motivation
Consistency

## Type of change
Choose one: Other

## Checklist

- [x] I have performed a self-review of my code.
- [x] I have added detailed comments to my code where applicable.
- [x] I have verified that my change does not break existing code.
- [x] My PR is based on the latest changes of the main branch (if unsure, please run `git pull --rebase upstream main`).
- [x] I am familiar with the [Google Style Guide](https://google.github.io/styleguide/) for the language I have coded in.
- [x] I have read through the [Contributing Guide](https://github.com/google/generative-ai-swift/blob/main/docs/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://cla.developers.google.com/about).

### CODE REVIEW

File: Tests/GoogleAITests/GenerateContentResponses/streaming-failure-malformed-content.txt [hunk 1]
```diff
@@ -1 +1 @@
...
+data: {"candidates": [{"content": {"missing-parts": true},"index": 0,"safetyRatings": [{"category": "HARM_CATEGORY_TOXICITY","probability": "NEGLIGIBLE"},{"category": "HARM_CATEGORY_SEXUALLY_EXPLICIT","probability": "NEGLIGIBLE"},{"category": "HARM_CATEGORY_VIOLENCE","probability": "NEGLIGIBLE"},{"category": "HARM_CATEGORY_HATE_SPEECH","probability": "NEGLIGIBLE"},{"category": "HARM_CATEGORY_HARASSMENT","probability": "NEGLIGIBLE"},{"category": "HARM_CATEGORY_DANGEROUS_CONTENT","probability": "NEGLIGIBLE"}]}],"promptFeedback": {"safetyRatings": [{"category": "HARM_CATEGORY_TOXICITY","probability": "NEGLIGIBLE"},{"category": "HARM_CATEGORY_SEXUALLY_EXPLICIT","probability": "NEGLIGIBLE"},{"category": "HARM_CATEGORY_VIOLENCE","probability": "NEGLIGIBLE"},{"category": "HARM_CATEGORY_HATE_SPEECH","probability": "NEGLIGIBLE"},{"category": "HARM_CATEGORY_HARASSMENT","probability": "NEGLIGIBLE"},{"category": "HARM_CATEGORY_DANGEROUS_CONTENT","probability": "NEGLIGIBLE"}]}}
```
Comment (paulb777) [inline Tests/GoogleAITests/GenerateContentResponses/streaming-failure-malformed-content.txt:1]: Is the change from "invalid-field" to "missing-parts" intentional?

File: Tests/GoogleAITests/GenerativeModelTests.swift [hunk 2]
```diff
@@ -400,7 +385,15 @@ final class GenerativeModelTests: XCTestCase {
...
-    try await internalTestGenerateContent(resource: "MissingSafetyRatings", safetyRatingsCount: 0)
+    MockURLProtocol.requestHandler = try httpRequestHandler(
+      forResource: "unary-success-missing-safety-ratings",
+      withExtension: "json"
+    )
+
+    let content = try await model.generateContent(testPrompt)
+    let promptFeedback = try XCTUnwrap(content.promptFeedback)
+    XCTAssertEqual(promptFeedback.safetyRatings.count, 0)
+    XCTAssertNotNil(content.text)
```
Comment (paulb777) [inline Tests/GoogleAITests/GenerativeModelTests.swift:396]: Why not an `XCTAssertEqual` here?

**Review by paulb777 — APPROVED** [review]

File: Tests/GoogleAITests/GenerateContentResponses/streaming-failure-malformed-content.txt (hunk 1 above)
Comment (ryanwilson) [reply Tests/GoogleAITests/GenerateContentResponses/streaming-failure-malformed-content.txt:1]: Yes - I was originally confused and thought a new field should cause an error, but this better calls out that something expected is missing. Can revert or update if you think something is better

Comment (paulb777) [reply Tests/GoogleAITests/GenerateContentResponses/streaming-failure-malformed-content.txt:1]: SGTM

File: Tests/GoogleAITests/GenerativeModelTests.swift (hunk 2 above)
Comment (ryanwilson) [reply Tests/GoogleAITests/GenerativeModelTests.swift]: Updated, the text was quite long in the test file so I've  shortened it and added an `XCTAssertEqual`

**Pull request merged**